src/index.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
 import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] As I can see you imported calculator inside index.js file but as required you should import it inside app.js and put all your components there in app.js because inside index.js it should be just app compnent because this is the flow of react and you should follow it 💯🚀🔥😊

## and please remeber to make your app.js based class 💯💯
![image](https://user-images.githubusercontent.com/58553711/199625084-4ade37f9-840f-4d40-a4a0-5febc875038f.png)